### PR TITLE
Bump version to 7.2.0

### PR DESCRIPTION
**Pull request by sebastienlagarde**
Bump version to 7.2.0

### CODE REVIEW

File: com.unity.testframework.graphics/package.json [hunk 1]
```diff
@@ -1,9 +1,9 @@
 {
     "name": "com.unity.testframework.graphics",
     "displayName":"Graphics Tests Framework",
-    "version": "7.1.1",
+    "version": "7.2.0",
     "unity": "2019.3",
-    "unityRelease": "0a12",
+    "unityRelease": "0b13",
```
Comment (sophiaaar) [inline com.unity.testframework.graphics/package.json]: are these changes intended? 0b13 isn't out yet

Comment (Verasl) [reply com.unity.testframework.graphics/package.json]: should be b3

Comment (sebastienlagarde) [reply com.unity.testframework.graphics/package.json]: right, fixed!
